### ISSUE TITLE
mkl_jll hangs during precompilation

### ISSUE THREAD

**Issue by ufechner7** [JuliaPackaging/Yggdrasil#9281]
Installing LinearSolve.jl results in:
```
Precompiling project...
  227 dependencies successfully precompiled in 96 seconds
  1 dependency had output during precompilation:
┌ MKL_jll
│   Downloading artifact: MKL
│  
│  [pid 75780] waiting for IO to finish:
│   Handle type        uv_handle_t->data
│   timer              0x23375d0->0x71b7ed8cea70
│  This means that a package has started a background task or event source that has not finished running. For precompilation to complete successfully, the event source needs to be closed explicitly. See the developer documentation on fixing precompilation hangs for more help.
│  
│  [pid 75780] waiting for IO to finish:
│   Handle type        uv_handle_t->data
│   timer              0x23375d0->0x71b7ed8cea70
│  This means that a package has started a background task or event source that has not finished running. For precompilation to complete successfully, the event source needs to be closed explicitly. See the developer documentation on fixing precompilation hangs for more help.
└  
```
Can this be fixed here?

see also: https://github.com/SciML/LinearSolve.jl/issues/524

**Comment by giordano** [comment]
Not related to this repository, and already fixed by https://github.com/JuliaLang/Pkg.jl/pull/3950

**Comment by ufechner7** [comment]
In which Julia version is this fixed?

**Comment by ufechner7** [comment]
Script to reproduce the issue:
```
#!/bin/bash -eu
rm -rf /tmp/testdepot
mkdir /tmp/testdepot
rm -rf /tmp/test
mkdir /tmp/test
cd /tmp/test
export JULIA_DEPOT_PATH=/tmp/testdepot 
julia --project="." -e "using Pkg; Pkg.add(\"LinearSolve\"); using LinearSolve"
cd ..
```

**Comment by giordano** [comment]
> In which Julia version is this fixed?

1.11 and probably 1.10.5 too.

> It is not fixed in Julia 1.10.4. Can you please re-open this issue?

No, why I should? This has nothing to do with this repository, it's a Pkg bug, and has been already fixed.
